dagmc_stats/DagmcFile.py

```diff
@@ -106,6 +109,50 @@ def __set_entityset_ranges(self):
                 self._my_moab_core.get_entities_by_type_and_tag(self.root_set, types.MBENTITYSET,
                                                                 self.dagmc_tags['geom_dim'], [dimension])
 
+    def get_meshset_by_id(self, dim, ids=[]):
+        """Get meshset of the geometry with specified dimension and ids
+
+        inputs
+        ------
+        dim : (Integer or String) Dimension of the meshset. 0: 'node(s)',
+                                1: 'curve(s)', 2: 'surface(s)', 3: 'volume(s)'
+        ids : (Integer) Global ID(s) of the meshset
+
+        outputs
+        -------
+        meshset : meshset of the geometry with given dimension and ids. First,
+                  dim will be checked. If dim is invalid, the root set will be
+                  returned. Then, if id is invalid (empty or not in the given
+                  dim range), all entities with the given dim will be returned.
+        """
+        if isinstance(dim, int) and dim in [0, 1, 2, 3]:
```

Comment:
Connect this to the definition of this range of valid values

```suggestion
        if isinstance(dim, int) and dim in self.entityset_types.keys():
```